acceptance/gossip_peerings_test.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/cockroachdb/cockroach/gossip"
 	"github.com/cockroachdb/cockroach/util"
 	"github.com/cockroachdb/cockroach/util/log"
+	"github.com/cockroachdb/cockroach/util/randhelper"
```

Comment:
Not used to seeing the term `helper` in a package name. Perhaps `s/randhelper/randutil/g`?